features/ui/footer/footer.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import styled from "styled-components";
+import { color, space, breakpoint, textFont } from "@styles/theme";
+import Link from "next/link";
+
+const Container = styled.footer`
+  background-color: ${color("gray", 50)};
+  height: 11rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+  padding-inline: 0;
+
+  @media (min-width: ${breakpoint("desktop")}) {
+    flex-direction: row;
+    justify-content: space-between;
+    height: 3.7rem;
+    padding-inline: ${space(8)};
+  }
+`;
+
+const Version = styled.p`
+  color: ${color("gray", 400)};
+  order: 3;
+  padding: 0;
+  margin: 0;
+
+  @media (min-width: ${breakpoint("desktop")}) {
+    order: 0;
+    padding: initial;
+    margin: initial;
+  }
+`;
+
+const Nav = styled.div`
+  margin-right: ${space(16)};
+  order: 1;
+  margin: 0 auto;
+
+  @media (min-width: ${breakpoint("desktop")}) {
+    order: 0;
+    margin-right: 0;
+  }
+`;
+
+const List = styled.ul`
+  margin: 0;
+  padding: 0;
+  display: flex;
+  gap: ${space(6)};
+  list-style-type: none;
+  width: 100%;
+
+  @media (min-width: ${breakpoint("desktop")}) {
+    width: 18rem;
+  }
+`;
+
+const ListItem = styled(Link)`
+  display: inline-block;
+`;
+
+const Anchor = styled.a`
+  display: inline-block;
+  color: ${color("gray", 500)};
+  font: ${textFont("sm", "regular")};
+`;
+
+const FooterIcon = styled.img`
+  display: block;
+  width: ${space(6)} - 0.25;
+  order: 2;
+
+  @media (min-width: ${breakpoint("desktop")}) {
+    order: 0;
+  }
+`;
+
+export function Footer() {
+  return (
+    <Container>
+      <Version>Version: 14.5.1</Version>
```

Comment:
I think the task wasn't totally clear here. There's an acceptance criterium "The current version number of the application is shown". This was meant to be dynamic depending on the actual version of the app and not hard-coded. So the question is where do you get that number from?

Reply:
🤔 I got the number from the Figma design file. Is there an API endpoint to retrieve the latest version? Perhaps you just meant it should be coded in order to be ready to receive dynamic data, maybe only showing the static number if no active data is currently available(with a conditional).